source/includes/harvest/_jobs.md

```diff
@@ -338,4 +338,135 @@ Parameter | Description
 id | The ID of the job to retrieve
 
 <br>
-[See noteworthy response attributes.](#the-job-object)
\ No newline at end of file
+[See noteworthy response attributes.](#the-job-object)
+
+## Update a job
+
+```shell
+curl --request PATCH 'https://harvest.greenhouse.io/v1/jobs/{id}'
+-H "Authorization: Basic MGQwMzFkODIyN2VhZmE2MWRjMzc1YTZjMmUwNjdlMjQ6"
+```
+
+```json
+{
+   "id": 144371,
+   "name": "New job name",
+   "requisition_id": "1",
+   "notes": "Here are some notes",
+   "status": "open",
+   "opened_at": "2015-11-19T19:53:32.662Z",
+   "team_and_responsibilities": "Info",
+   "how_to_sell_this_job": "snacks",
+   "original_job_id": 3,
```

Comment:
We should probably add a description of what this thing is?  Is this like a job id that it was copied from?  If it is, we probably shouldn't let them update it because it's historical data.